tests/operators/test_agnostic_load_file.py

```diff
@@ -50,6 +51,29 @@
 
 log = logging.getLogger(__name__)
 DEFAULT_DATE = timezone.datetime(2016, 1, 1)
+OUTPUT_TABLE_NAME = test_utils.get_table_name("load_file_test_table")
+OUTPUT_SCHEMA = os.getenv("SNOWFLAKE_SCHEMA")
+CWD = pathlib.Path(__file__).parent
+
+DEFAULT_SCHEMA = "tmp_astro"
+SQL_SERVER_HOOK_PARAMETERS = {
+    "snowflake": {
+        "snowflake_conn_id": "snowflake_conn",
+        "schema": os.getenv("SNOWFLAKE_SCHEMA"),
+        "database": os.getenv("SNOWFLAKE_DATABASE"),
+        "warehouse": os.getenv("SNOWFLAKE_WAREHOUSE"),
+    },
+    "postgres": {"postgres_conn_id": "postgres_conn"},
```

Comment:
This looks much better. Much easier to add future DBs. I'd love to know if there was a way for us to set a flag with pytest to only run postgres or only run snowflake?

Reply:
@dimberman, this is an example if we want to run the `ndjson` + `snowflake` combination:
```
pytest  -v -k "test_load_file[ndjson-snowflake]"
```
And this is another example if we want to run all the tests which have `snowflake` - either in their name, or as a parameter from parametrize:
```
pytest  -v -k "test_load_file" -k snowflake
```
https://docs.pytest.org/en/6.2.x/example/markers.html#using-k-expr-to-select-tests-based-on-their-name